src/custom/CortiAuth.ts

```diff
@@ -116,21 +200,45 @@ export class Auth extends FernAuth {
         }, requestOptions));
     }
 
+    /**
+     * Patch: PKCE-specific method for Authorization code flow
+     */
+    public getPkceFlowToken(
+        request: AuthorizationPkceServer,
+        requestOptions?: FernAuth.RequestOptions,
+    ): core.HttpResponsePromise<Corti.GetTokenResponse> {
+        const codeVerifier = request.codeVerifier || this.getCodeVerifier();
```

Comment:
Why would we generate code verifier by default? They must pass it, otherwise they should another authentication method